specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/examples/ApiManagementListProductGroups.json

```diff
@@ -4,22 +4,44 @@
         "resourceGroupName": "rg1",
         "api-version": "2017-03-01",
         "subscriptionId": "subid",
-        "productId": "unlimited"
+        "productId": "5600b57e7e8880006a060002"
     },
     "responses": {
         "200": {
             "body": {
                 "value": [
                     {
-                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/groups/administrators",
+                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/products/5600b57e7e8880006a060002/groups/5600b57e7e8880006a020001",
                         "type": "Microsoft.ApiManagement/service/products/groups",
-                        "name": "administrators",
+                        "name": "5600b57e7e8880006a020001",
```

Comment:
Just out of curiosity, this is just an example and not an actual name, correct?

Reply:
yes, customer can choose to provide a name or not.

If he does not, then we create for customer. The Name which is actually associated with the entity is "displayName" which can have spaces and is not restricted by naming conventions of URI pattern.

---
In reply to: [147819237](https://github.com/Azure/azure-rest-api-specs/pull/1945#discussion_r147819237) [](ancestors = 147819237)

Reply:
I understand. What concerns me is whether this is some sort of a service side secret. If it isn't, no issues

Reply:
No secret.